django_project/changes/views/version.py

```diff
@@ -85,6 +93,28 @@ def get_context_data(self, **kwargs):
         if project_slug:
             context['the_project'] = Project.objects.get(slug=project_slug)
             context['project'] = context['the_project']
+
+        # lets check for specific user permissions here.
+        if self.request.user.is_authenticated:
+            context['user_can_view'] = True
+
+        if self.request.user.is_staff:
+            context['user_can_edit'] = True
+            context['user_can_delete'] = True
+
+        if self.request.user.is_superuser:
+            context['user_can_edit'] = True
+            context['user_can_delete'] = True
+
+        version_managers = (
+            context['project'].changelog_managers.all())
+        if self.request.user in version_managers:
+            context['user_can_edit'] = True
+            context['user_can_delete'] = True
+
+        if self.request.user == context['project'].owner:
+            context['version_owner'] = True
```

Comment:
don't you want / need to add can_edit / can_delete here?

Reply:
yes please.
Don't introduce this new variable ['version_owner']. Just override to True can_edit, can_delete, can_create

Reply:
Yes to that makes sence to reduce repeatition as well.
Updated.